packages/widgets-internal/components/TokenFilter/TokenFilter.tsx

```diff
@@ -61,18 +61,19 @@ const ItemName = styled.span`
   font-weight: 400;
 `;
 
-export const toTokenValue = (t: Currency) => `${t.chainId}:${t instanceof Token ? t.address : zeroAddress}`;
+export const toTokenValue = (t: Currency) => `${t.chainId}:${t.isNative ? t.wrapped.address : t.address}`;
```

Comment:
just use `t.wrapped.address` is ok, if t is a Token, the wrapped token is itself

Reply:
done